CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 8.7.1
```

Comment:
@chandwanitulsi There were a couple of more security dependency updates pending which I just merged. Could you please include those.

Reply:
Yes of course, Thanks for merging